src/app/_components/List/_bListItem.tsx

```diff
@@ -26,10 +26,12 @@ const BlindBoxItem: React.FC<BlindBoxItemProps> = ({ blindBox, isSelected, onSel
       onClick={interactionType && interactionType > 1 ? onSelect : undefined}
     >
       <div className={`${viewMode === 'list' ? 'w-12 h-12 mr-4' : 'h-[120px]'} relative`}>
-        <img alt={blindBox.id!} src={`/svg/BlindBox.svg`} className="rounded max-h-[120px] object-cover w-full"/>
+        <Image alt={blindBox.id!} src={`/svg/blindbox-animation-${Math.ceil(Math.random()*6)}.svg`} className="rounded" layout='fill' objectFit='cover'/>
```

Comment:
- I think after assigning a color to a blind box, we should maintain consistent coloring to avoid confusion and enhance user recognition. 
- The gray one should only be assigned to 
<img width="449" alt="Screenshot 2024-01-24 at 9 33 56 AM" src="https://github.com/lee920217/PhilosopherStone/assets/140578792/e4203b0f-ac3e-4dfc-8406-004ea7ebbab3">
an empty blind box